packages/ckeditor5-coremedia-link/src/linktarget/LinkTargetModelView.ts

```diff
@@ -34,6 +39,47 @@ export default class LinkTargetModelView extends Plugin {
 
     editor.commands.add("linkTarget", new LinkTargetCommand(editor));
 
+    // This function, as well as the postFixer are used to apply default link targets to links in the editor.
+    const addLinkTarget = (linkTarget: string, range: Range) => {
+      this.editor.model.change((writer) => {
+        writer.setAttribute("linkTarget", linkTarget, range);
+      });
+    };
+
+    this.editor.model.document.registerPostFixer((writer) => {
+      const changes = this.editor.model.document.differ.getChanges();
+
+      for (const entry of changes) {
+        if (entry.type === "attribute" && entry.attributeKey === "linkHref") {
```

Comment:
Maybe a method can be extracted which can be unit tested. 
The result would be the linkTarget, the range must be evaluated afterwards, the addLinkTarget call would be afterwards. Not sure if it is that easy.